City-of-Turku/kada
# Feature/docker performance

To test:

1) take backup of the database
2) stop and remove docker containers (note: you'll lose the whole container incl db):
     docker-compose down
3) Create and start containers:
     docker-compose up
4) Verify that your mounted volume is working (files are accessible both in host and inside container)
     docker-compose exec --user drupal drupal bash
     ls -lh
